tests/main.test.ts

```diff
@@ -195,51 +195,41 @@ describe("User start/stop", () => {
     }
   });
 
-  test("User can't start another issue if they have reached the max limit", async () => {
-    jest.mock("../src/utils/issue", () => ({
-      getAvailableOpenedPullRequests: jest.fn().mockResolvedValue([
-        {
-          number: 1,
-          reviews: [
-            {
-              state: "APPROVED",
-            },
-          ],
-        },
-        {
-          number: 2,
-          reviews: [
-            {
-              state: "APPROVED",
-            },
-          ],
-        },
-        {
-          number: 3,
-          reviews: [
-            {
-              state: "APPROVED",
-            },
-          ],
-        },
-      ]),
-    }));
+  test("should return the role with the smallest task limit if user role is not defined in config", async () => {
 
-    const issue = db.issue.findFirst({ where: { id: { equals: 1 } } }) as unknown as Issue;
+    const issue = db.issue.findFirst({ where: { id: { equals: 8 } } }) as unknown as Issue;
     const sender = db.users.findFirst({ where: { id: { equals: 1 } } }) as unknown as Sender;
 
     const context = createContext(issue, sender);
 
-    context.adapters = createAdapters(getSupabase(), context as unknown as Context);
+    context.adapters = createAdapters(getSupabase(), context);
 
     try {
-      await userStartStop(context as unknown as Context);
+      await userStartStop(context);
     } catch (error) {
       if (error instanceof Error) {
         expect(error.message).toEqual("Too many assigned issues, you have reached your max limit of 3 issues.");
       }
     }
   });
+
+  test("should set maxLimits to 5 if the user is a member", async () => {
+
+    const issue = db.issue.findFirst({ where: { id: { equals: 7 } } }) as unknown as Issue;
+    const sender = db.users.findFirst({ where: { id: { equals: 1 } } }) as unknown as Sender;
```

Comment:
This doesn't feel right.

`sender === 1 === "ubiquity" === "admin"`

If sender is an admin and they are sending the `/start` command then the error below should match the config value which is `10`